### DISCUSSION TITLE
What can I do to unblock using crosh?

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1241] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
What can I do to unblock using crosh?

**Comment by Balistic123** [comment]
if it says crosh has been blocked by administrator then it is blocked by policy and you can't do anything about it otherwise there are probably solutions

**Reply by Sam-hacks1** [reply]
no to remove extensions using crosh

**Reply by no-name-6655** [reply, marked as answer]
No, maybe you can try finding a way but I'm pretty sure crosh has no way of managing installed extensions.
